music21/musicxml/testPrimitive.py

```diff
@@ -2724,7 +2724,7 @@
           <ornaments>
             <turn/>
             <accidental-mark placement="above">sharp</accidental-mark>
-            <accidental-mark placement="above">three-quarters-flat</accidental-mark>
+            <accidental-mark placement="below">flat</accidental-mark>
```

Comment:
good catch, but can we change the mark itself to three-quarters-flat?  It's an important test of ability to do microtones.

Reply:
I changed this so that it would be something Finale was happy to render (so I could see it during debugging).  But I'm fine to change it back once I have things working again.

Reply:
Still need to do this.

Reply:
Done.